example-1/script.js

```diff
@@ -120,9 +118,12 @@ form.addEventListener("submit", function (event) {
     var token = result.token;
     var error = result.error;
 
+    formError.classList.remove("has-error");
```

Comment:
I would move this above the `if` statement, because we will always be in this block if `event.error` is defined